app/admin.py

```diff
@@ -130,6 +128,8 @@ def post(self):
                     ],
                     # These settings are literal strings (not JSON).
                     literal_config_names=[
+                        'unreviewed_notes_email',
```

Comment:
Optional: If you plan to use this for other notifications in the future, you may want to use more general name e.g., notification_email here.